pkgs/top-level/stage.nix

```diff
@@ -79,11 +79,20 @@ let
 
   # The old identifiers for cross-compiling. These should eventually be removed,
   # and the packages that rely on them refactored accordingly.
-  platformCompat = self: super: let
-    inherit (super.stdenv) buildPlatform hostPlatform targetPlatform;
-  in {
-    inherit buildPlatform hostPlatform targetPlatform;
-    inherit (buildPlatform) system;
+  platformCompat = self: super: {
+    buildPlatform = builtins.trace
```

Comment:
I'm not sure if we need to deprecate these. To me it seems like the difference between using ```lib`` or ```stdenv.lib```. No real benefits for one over the other.

Reply:
I agree these deprecations, unlike the others, are "fully explicit" so there's less a downside. But still, I think it benefits no one to have redundant bindings---I'd actually be for deprecating one of `lib` or `stdenv.lib` too for the same reason.